src/main/java/com/ericsson/eiffel/remrem/publish/service/MessageService.java

```diff
@@ -11,7 +12,7 @@
      * @param msgs the list with messages to be sent
      * @return list of results containing send response messages after sending the messages 
      */
-    List<SendResult> send(String routingKey, List<String> msgs);
+    SendResult send(Map<String, String> routekeyMap, Map<String, String> msgs);
```

Comment:
The changes has not been reflected in the documentation above. Also use camelcase for routekeyMap